File: frontend/src/types/venta.ts
export interface Cliente {
    id_cli: number;
    nom_cli: string;
    ap_pat_cli: string;
    ap_mat_cli: string;
    ci_cli: string;
    cod_cli?: string;
}

export interface Empleado {
    id_emp: number;
    nom_emp: string;
    ap_pat_emp?: string;
    ap_mat_emp?: string;
    cod_emp?: string;
}

export interface Venta {
    id_ven: number;
    cod_ven?: string;
    fec_ven: string;
    est_ven: string;
    total_ven: number;
    descuento: number;
    notas: string;
    id_cli: number;
    cliente?: Cliente;
    id_emp: number;
    empleado?: Empleado;
}

File: backend/src/main/java/com/changuitostudio/backend/domain/model/Venta.java
package com.changuitostudio.backend.domain.model;

import java.time.LocalDate;

public class Venta {
    private Long id;
    private LocalDate fecVen;
    private String estVen;
    private Double totalVen;
    private Double descuento;
    private Cliente cliente;
    private Empleado empleado;
    private String notas;
    private String codVen;

    public Venta() {
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }
    public LocalDate getFecVen() {
        return fecVen;
    }

    public void setFecVen(LocalDate fecVen) {
        this.fecVen = fecVen;
    }
    public String getEstVen() {
        return estVen;
    }

    public void setEstVen(String estVen) {
        this.estVen = estVen;
    }
    public Double getTotalVen() {
        return totalVen;
    }

    public void setTotalVen(Double totalVen) {
        this.totalVen = totalVen;
    }
    public Double getDescuento() {
        return descuento;
    }

    public void setDescuento(Double descuento) {
        this.descuento = descuento;
    }
    public Cliente getCliente() {
        return cliente;
    }

    public void setCliente(Cliente cliente) {
        this.cliente = cliente;
    }
    public Empleado getEmpleado() {
        return empleado;
    }

    public void setEmpleado(Empleado empleado) {
        this.empleado = empleado;
    }
    public String getNotas() {
        return notas;
    }

    public void setNotas(String notas) {
        this.notas = notas;
    }
    public String getCodVen() {
        return codVen;
    }

    public void setCodVen(String codVen) {
        this.codVen = codVen;
    }
}
